.buildpacks

```diff
@@ -0,0 +1,2 @@
+https://github.com/Scalingo/ruby-buildpack
```

Comment:
we could switch the dockerfile to use ubuntu/debian rather than alpine if it would make things easier?

Reply:
Hi Andrew, I don't think dockerfile is used in herokuish build. I think it's more an ansible-dokku issue (and it would be easier if ansible-dokku could set buildpack directly).

I'm totally open to suggestion about this subject, it's better to not stuff the project with config for optional third parties.